alws/crud/test_repository.py

```diff
@@ -139,6 +168,44 @@ async def bulk_create_package_mapping(
     await session.flush()
 
 
+def get_team_role_name(team_name: str, role_name: str):
+    return f'{team_name}_{role_name}'
+
+async def create_test_repository_role_mapping(session: AsyncSession, team_name: str):
+    required_roles = (Contributor,)
+    required_actions = (UpdateTest, DeleteTest)
+    role_names = [get_team_role_name(team_name, role.name) for role in required_roles]
+
+    existing_roles = (
+        await session.execute(
+            select(UserRole).where(
+                UserRole.name.in_(role_names)
+            ).options(
+                selectinload(UserRole.actions)
+            )
+        )
+    ).scalars().all()
+    # existing_role_names = {r.name for r in existing_roles}
```

Comment:
Please delete this and other irrelevant comments

Reply:
Done, I checked all of the other files, I believe I've removed them all.